Packs/RSTCloud/Playbooks/IP_Enrichment_-_External_-_RST_Threat_Feed.yml

```diff
@@ -0,0 +1,554 @@
+id: IP Enrichment - External - RST Threat Feed
+version: -1
+fromversion: 5.5.0
```

Comment:
Why is this fromversion 5.5.0 while the integration is 5.0.0?

Reply:
updated to be 5.0.0